edx_repo_tools/utils.py

```diff
@@ -45,7 +45,8 @@ class YamlLoader:
     def __init__(self, file_path):
         self.file_path = file_path
         self.yml_instance = YAML()
-        self.yml_instance.indent(mapping=2, sequence=4, offset=2)
+        self.yml_instance.default_flow_style = None
+        self.yml_instance.indent(mapping=2, sequence=2, offset=0)
```

Comment:
LGTM apart from one change. Could you please remove the local indentation related settings like [this](https://github.com/openedx/repo-tools/blob/master/edx_repo_tools/codemods/django3/github_actions_modernizer.py#L18) from other modernizers as due to this change they all become just extraneous. Anyway this isn't a blocker and I will approve the changes.

Reply:
Yes I'll made those changes in a separate PR as they don't come under the scope of this PR